src/app/util/form.validators.ts

```diff
@@ -120,8 +119,16 @@ function notOnlyWhitespace(control: AbstractControl) {
     return text.trim().length <= 0 ? {'onlyWhitespace': true} : null;
 }
 
+function isNumber(control: AbstractControl) {
```

Comment:
Vielleicht wäre es gut, wenn wir solche Funktionen in eine Util Klasse packen.

Reply:
Das ist doch unter app/util.